core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -261,6 +261,32 @@ public static boolean isLiteralChain(SqlNode node) {
     }
   }
 
+  /**
+   * Unparses a call to an operator which has special syntax.
+   *
+   * @param operator The operator
+   * @param writer   Writer
+   * @param call     List of 0 or more operands
+   */
+  public static void unparseSpecialSyntax(SqlOperator operator,
+      SqlWriter writer,
+      SqlCall call) {
+    final List<SqlNode> operands = call.getOperandList();
+    switch (operator.getKind()) {
+    case INSERT:
+      assert operands.size() == 4;
+      final SqlInsert insert = new SqlInsert(call.getParserPosition(),
+          (SqlNodeList) operands.get(0), operands.get(1), operands.get(2),
+          operands.get(3) == null ? null : (SqlNodeList) operands.get(3));
```

Comment:
Just use `(SqlNodeList) operands.get(3)` will be fine

Reply:
ok, Thanks a lot.